### PR TITLE
fix(convert): Correct Conversion of o3

### LINKED ISSUE #48: Bad Data Returned from US EPA Ozone to Raw
It looks like your code uses the 1 hour sampling of Ozone when converting from US EPA. However, aqicn (and maybe others) are returning the 8 hour sampling, so it results in a 0 being returned.

### PR DESCRIPTION
**Pull request by Sunoo**
fixes #48 

The formula for converting o3 8h readings divides the raw value by 1000. To compensate for this, multiply each of the breakpoints for this pollutant by 1000.

### CODE REVIEW

**Comment by Sunoo** [comment]
Looks like the only failures now are from missing tokens calling aqicn, which is code I didn't touch, so I'm assuming it can be safely ignored.

**Comment by amaurymartiny** [comment]
Wow, thanks so much for these 2 PRs! I just scanned through them, they look very nice, I'll have a closer look tomorrow.

Yeah, you can forget about the AQICN token errors for now,  I'll clean up those tomorrow.

**Comment by Sunoo** [comment]
No problem, I'm glad I could give back to a library I found helpful.

File: packages/convert/src/aqi/usaEpa/usaEpa.spec.ts [hunk 1]
```diff
@@ -16,16 +16,16 @@ describe('Convert pm25', () => {
...
-  testConvert('usaEpa', 'o3', 122, 0.077);
+  testConvert('usaEpa', 'o3', 122, 76.9);
 
   // From https://github.com/hrbonz/python-aqi/blob/master/test/test_epa.py
   testConvert('usaEpa', 'pm25', 39, 9.3);
   testConvert('usaEpa', 'pm25', 57, 15);
   testConvert('usaEpa', 'pm25', 135, 49.5);
   testConvert('usaEpa', 'pm25', 285, 235.4);
-  testConvert('usaEpa', 'o3', 155, 0.08753333); // Note: different from https://github.com/hrbonz/python-aqi/blob/1de807365eb44ef6c4e3d80af500ff8f6c273d41/test/test_epa.py#L34-L45
+  testConvert('usaEpa', 'o3', 155, 87.5); // Note: different from https://github.com/hrbonz/python-aqi/blob/1de807365eb44ef6c4e3d80af500ff8f6c273d41/test/test_epa.py#L34-L45
```
Comment (amaurymartiny) [inline packages/convert/src/aqi/usaEpa/usaEpa.spec.ts]: I see, so 87.5 here is `ppb` 👍

**Pull request merged**